source/MRViewer/MRMenu.cpp

```diff
@@ -792,16 +792,18 @@ void Menu::draw_helpers()
         ImGui::End();
     }
 
-    if ( show_rename_modal_ )
-    {
+    if ( show_rename_modal_  && !ImGui::IsPopupOpen( "Rename object" ) )
         ImGui::OpenPopup( "Rename object" );
```

Comment:
i think here we can just
```c++
if ( show_rename_modal_  )
{
    ImGui::OpenPopup( "Rename object" );
    show_rename_modal_  = false;
}
```
and do not reset `show_rename_modal_ ` in popup code

Reply:
(this will not work with `storedError_`)